engine/schema/src/main/java/com/cloud/service/dao/ServiceOfferingDaoImpl.java

```diff
@@ -58,7 +62,7 @@ public ServiceOfferingDaoImpl() {
 
         UniqueNameSearch = createSearchBuilder();
         UniqueNameSearch.and("name", UniqueNameSearch.entity().getUniqueName(), SearchCriteria.Op.EQ);
-        UniqueNameSearch.and("system", UniqueNameSearch.entity().isSystemUse(), SearchCriteria.Op.EQ);
+        UniqueNameSearch.and("system_use", UniqueNameSearch.entity().isSystemUse(), SearchCriteria.Op.EQ);
```

Comment:
As it will be used more times in this class, could it be extracted to a variable?